unic/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "unic"
+version = "0.4.0"
+authors = ["The UNIC Project Developers"]
+homepage = "https://github.com/behnam/rust-unic/"
+repository = "https://github.com/behnam/rust-unic/"
+license = "MIT/Apache-2.0"
+keywords = ["text", "unicode"]
+description = "UNIC - Unicode and Internationalization Crates"
+categories = ["parsing", "rendering", "encoding", "development-tools"]
+readme = "README.md"
+
+exclude = [
+    "data/**",
+    "tools/**",
+    ".cargo/**",
```

Comment:
These three exclusion lines don't mean anything anymore, since this Cargo.toml is below the point where it can see the directories `data`, `tools` and `.cargo`.